packages/http-client-python/emitter/src/http.ts

```diff
@@ -126,23 +252,24 @@ function addPagingInformation(
     discriminator: "paging",
     exposeStreamKeyword: false,
     itemName,
-    continuationTokenName,
+    nextLinkName,
     itemType,
     description: method.doc ?? "",
     summary: method.summary,
+    continuationTokenRequest: buildContinuationToken(
+      context,
+      method.pagingMetadata.continuationTokenParameterSegments,
+      method,
+    ),
+    continuationTokenResponse: buildContinuationToken(
+      context,
+      method.pagingMetadata.continuationTokenResponseSegments,
+      method,
+      false,
+    ),
```

Comment:
can we make it to a form and do it all in the `buildContinuationToken`? too many duplications here.

Reply:
Updated